### PR TITLE
release 0.0.11 - bug fixes and enhancements

### PR DESCRIPTION
**Pull request by kubawach**
(See changelog)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
 ## [Unreleased]
 
+## [0.0.10] - 2021-10-06
+
+### Bugfixes
+- Fixed issue with AWS lambda logging dependencies not included if the wrapper is used as a direct dependency
+
+### Enhancements
+- Added info message if logging exporter is installed for debug purposes
+- Default value of the 1configuration property `otel.lib.log.level` set to `INFO`
```
Comment (laurit) [inline CHANGELOG.md:17]: Is `1configuration` a typo?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.0.10] - 2021-10-06
```
[the remark this answers is not in the document]
Comment (kubawach) [reply CHANGELOG.md:10]: Ahh copy-paste syndrome :D

File: CHANGELOG.md (hunk 1 above)
Comment (kubawach) [reply CHANGELOG.md:17]: Yep!

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+    
+2. Ensuring that traces are ingested.
+
+    Wrapper will only wait configured (`OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT`) number of milliseconds for backend to ingest the spans. Increase the value if you don't see the spans in the APM.
+
+3. Verifying traces in the APM.
+
+   Setting `OTEL_LAMBDA_LOG_LEVEL` to `DEBUG` will install logging exporter along with the configured one. Logging exporter will, in turn, log every span including traceId and spanId. TraceId can be then used in the APM to check if the trace was properly ingested. 
+   
+   The setting will also cause `jaeger-thrift-exporter` (if used) to log each trace send to APM along with the backend URL and token (masked) used by the operation.
```
[the remark this answers is not in the document]
Comment (kubawach) [reply TROUBLESHOOTING.md:17]: Thanks!

**Pull request merged**